src/net/offline_fluereflows.rs

```diff
@@ -28,16 +28,16 @@ pub async fn fluereflow_fileparse(arg: Args) -> Result<()> {
     let use_mac = arg.parameters.use_mac.ok_or_else(|| anyhow!("Use MAC parameter not found"))?;
     let _flow_timeout = arg.parameters.timeout.ok_or_else(|| anyhow!("Flow timeout parameter not found"))?;
     let verbose = arg.verbose.ok_or_else(|| anyhow!("Verbose parameter not found"))?;
-
-    let mut cap = Capture::from_file(file_name)
-        .map_err(|e| anyhow!("Failed to capture from file: {}", e))?;
-
-    let file_dir = "./output";
-    fs::create_dir_all(<&str>::clone(&file_dir))
-        .map_err(|e| anyhow!("Failed to create directory: {}", e))?;
-    if verbose >= 1 {
-        info!("Created directory: {}", file_dir);
-    }
+    let mut cap = Capture::from_file(file_name).map_err(|e| anyhow!("Failed to capture from file: {}", e))?; 
+    let mut file = fs::File::create(file_path.clone()).map_err(|e| anyhow!("Failed to create file: {}", e))?; 
+    info!("Received packet");
```

Comment:
change the Received Packet log to debug level

Reply:
the `Received packet`, `flow established` etc's are just log spam, change it to debug level logs